script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -725,12 +779,12 @@ The final memories of an unfamiliar someone――someone who secluded themselves
 [sha:081e061b6766e7f8b0153cac61a93affb27c4848]{
 -- Page 38, Offset 33210.
 -- ……手段がない。
-...There's no solution.
+There's no way out...
 }
 [sha:db482298044c8dfa8928f6641fbbc40793fc895e]{
 -- Page 38, Offset 33211.
 -- コイツは、俺が自殺すれば解決するという問題でもない。
-This is not something that can be solved by killing myself.
+This isn't something I can solve by killing myself.
```

Comment:
>There's nothing about this problem that can be solved by killing myself.

Reply:
Seems weirdly elaborated to me?

Reply:
It's to fit it into the structure of the line after and before.

If you can find a pattern that works for all of them where this one isn't a bit too long, then that would be preferrable.